src/convertToCSSPTValue.js

```diff
@@ -20,3 +22,26 @@ export default function convertToCSSPTValue(styleValue: string): number {
   }
   return value;
 }
+
+export function toClosestFontPtSize(styleValue: string): number {
+  const originalPTValue = convertToCSSPTValue(styleValue);
+
+  if (FONT_PT_SIZES.includes(originalPTValue)) {
+    return originalPTValue;
+  }
+
+  const largerValueIndex = FONT_PT_SIZES.findIndex(element => element > originalPTValue);
```

Comment:
you may use `reduce` to make the logic cleaner.

```
export function toClosestFontPtSize(styleValue: string): number {
  const originalPTValue = convertToCSSPTValue(styleValue);

  if (FONT_PT_SIZES.includes(originalPTValue)) {
    return originalPTValue;
  }

  return FONT_PT_SIZES.reduce((prev, curr) => {
    return Math.abs(curr - originalPTValue) < Math.abs(prev - originalPTValue) ? 
      curr :
      prev;
  });
}
```

Reply:
I can switch to this. We'll end up iterating through `FONT_PT_SIZE` every time, which is we can accept since it's pretty limited in options right now.